test/greenlight/test_test.clj

```diff
@@ -1,16 +1,18 @@
 (ns greenlight.test-test
   (:require
+    [clojure.java.io :as io]
     [clojure.test :refer :all]
     [com.stuartsierra.component :as component]
     [greenlight.step :as step]
     [greenlight.test :as test]
-    [greenlight.test-suite.blue :as blue]))
+    [greenlight.test-suite.blue :as blue]
+    [greenlight.test-suite.yellow :as yellow]))
 
 
 (deftest sample-test
   (let [system (component/system-map :greenlight.test-test/component 6)
         sample-test (blue/sample-test)
-        test-result (test/run-test! system sample-test)]
+        test-result (test/run-test! system {} sample-test)]
```

Comment:
The backwards-compatible arity would also save you from these no-op changes here.